Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -301,15 +307,15 @@ private void loadDataEditorSettings() {
      * @throws IOException
      *             if filesystem I/O fails
      */
-    private void openMetsFile() throws IOException, InvalidImagesException {
+    private boolean openMetsFile() throws IOException, InvalidImagesException {
```

Comment:
A `boolean` return value from `openMetsFile()` is confusing. Please change this back to `void`, and set `this.mediaUpdated` inside the function itself.

Reply:
Why this in confusing? A "open" method could or should return a boolean value except there are rules to use exception for different cases. Setting a class global state is a possible solution, right, but there are a lot of cases too where this state is not changed even a change has been performed.

Reply:
I agree with @henning-gerhardt that returning a boolean to signal whether a function was executed successfully or not is neither confusing nor unheard of. Many often used methods like `File.mkdir` do something similar.